sklearn/semi_supervised/_self_training.py

```diff
@@ -188,7 +192,7 @@ def __init__(
         # SelfTrainingClassifier.base_estimator is not validated yet
         prefer_skip_nested_validation=False
     )
-    def fit(self, X, y):
+    def fit(self, X, y, **fit_params):
```

Comment:
I would just call this `params` and be the same as all other methods.

Reply:
Done.